### PR TITLE
Create projects card

### PR DESCRIPTION
**Pull request by Mohammed-Sbeata**

### CODE REVIEW

**Review by nada-abuzaid — APPROVED** [review]

File: src/components/ProjectsCard/Project.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import {
```
Comment (nada-abuzaid) [inline src/components/ProjectsCard/Project.styled.tsx:1]: - change the file name `cards.styled.ts`
- no need to `style.ts` delete it please!

File: src/components/ProjectsCard/Project.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+} from '@mui/material';
+import DashboardIcon from '@mui/icons-material/Dashboard';
+
+export const Wrapper = styled(Card)(({ theme }) => ({
+  background: theme.palette.custom.black,
+  height: '100%',
+  display: 'grid',
+  gap: '52px',
+  gridTemplateColumns: 'auto auto auto',
+  padding: '10px',
```
Comment (nada-abuzaid) [inline src/components/ProjectsCard/Project.styled.tsx:16]: don't use pixel `px`

File: src/components/ProjectsCard/Project.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import {
+  styled,
+  Card,
+  Button,
+  CardContent,
+  Typography,
+} from '@mui/material';
+import DashboardIcon from '@mui/icons-material/Dashboard';
```
Comment (nada-abuzaid) [inline src/components/ProjectsCard/Project.styled.tsx:8]: replace it with the following icon from [react-icons](https://react-icons.github.io/react-icons/search?q=Dashboard)
```suggestion
import { MdDashboard } from 'react-icons/md';
```

**Review by nada-abuzaid — CHANGES_REQUESTED** [review]

File: src/components/ProjectsCard/Project.styled.tsx (hunk 1 above)
Comment (Mohammed-Sbeata) [reply src/components/ProjectsCard/Project.styled.tsx:1]: Ok, done

File: src/components/ProjectsCard/Project.styled.tsx (hunk 3 above)
Comment (Mohammed-Sbeata) [reply src/components/ProjectsCard/Project.styled.tsx:8]: ok, done.

File: src/components/ProjectsCard/Project.styled.tsx (hunk 2 above)
Comment (Mohammed-Sbeata) [reply src/components/ProjectsCard/Project.styled.tsx:16]: ok, done

**Review by nada-abuzaid — APPROVED** [review]

**Review by Amal-Mousa — APPROVED** [review]

**Pull request merged**